db/migrate/20150115155947_add_scaptimony_scap_content_digest.rb

```diff
@@ -0,0 +1,11 @@
+class AddScaptimonyScapContentDigest < ActiveRecord::Migration
+  def change
+    add_column :scaptimony_scap_contents, :digest, :string, :limit => 128
+    Scaptimony::ScapContent.find_each do |content|
```

Comment:
Updated, as you have wished.

Please note that upgrade of existing installation may fail anyway. That's because of d3ded5441cef4eaa573aefe57b2035d8afc20ee2.

In other words, we have had digest in 0.2.0 version and we create it again in 0.3.0 version. We do not support upgrades, I guess. Users (if any at this point) are advised to fresh install.